docs/platforms/android/session-replay/index.mdx

```diff
@@ -71,6 +71,16 @@ While you're testing, we recommend that you set `sessionSampleRate` to `1.0`. Th
 
 Once testing is complete, **we recommend lowering this value in production**. We still recommend keeping `onErrorSampleRate` set to `1.0`.
 
+## User Session
+
+A user session starts when the Sentry SDK is initialized or when the application enters the foreground. The session will capture screen transitions, navigations, touches and other events until the application is sent to the background. If the application is brought back to the foreground within 30 seconds (default), the same `replay_id` will be used and the session will continue. Otherwise, the session will be terminated according to the rules for [Release Health](/platforms/android/configuration/releases/#sessions). You can adjust the `sessionTrackingIntervalMillis` option to extend or shorten the duration of a single replay, depending on your needs.
+
+The session also has a maximum duration of 60 minutes. If the session is not terminated by the user, it will be terminated after 60 minutes.
```

Comment:
IMO this line should be part of this sentence: "Otherwise, the session will be terminated according to the rules for [Release Health](https://sentry-docs-ekn1li86v.sentry.dev/platforms/android/configuration/releases/#sessions)."

For example, "Otherwise, the session will be terminated ......., or when the maximum duration of 60 minutes is reached."